next/package.json

```diff
@@ -67,14 +67,14 @@
     "@tailwindcss/aspect-ratio": "^0.4.2",
     "@tailwindcss/typography": "^0.5.10",
     "@types/node": "^18.18.12",
```

Comment:
(Since we're using node 20, makes sense to update `@types/node` to v20 too)

Reply:
Good point! Updated in https://github.com/wunderio/next-drupal-starterkit/pull/198/commits/367e0ebf31327c60c37fd37c12e90e4cb3d73ec7